agasc/supplement/magnitudes/mag_estimate_report.py

```diff
@@ -107,6 +109,16 @@ def single_star_html(self, agasc_id, directory,
                                            obs_stats=o.as_array(),
                                            static_dir=static_dir,
                                            glossary=GLOSSARY))
+        with open(directory / 'data.json', 'w') as json_out:
+            json.dump(
+                {
+                    'agasc_stats': self.agasc_stats[self.agasc_stats['agasc_id'] == agasc_id],
+                    'obs_stats': o,
```

Comment:
This crosses the threshold of no banned single-letter variable names like `o` in new code. You can use VS code to rename this to `obj_stat` and the code will suddenly be more readable to me.

Reply:
There is an example output in the docstring, but if you want to see an actual file related to this PR, I can show you. These files are not indented so they are not so readable. I made a copy with indentation here: https://cxc.cfa.harvard.edu/mta/ASPECT/jgonzalez/agasc/example_data.json